app/page.tsx

```diff
@@ -99,113 +99,115 @@ export default function Home() {
+                        <p className={`${commonTextClass}`}>
+                          {formatSessionTime(data.sessionOccurrence.end_time)}
+                        </p>
+                      </div>
+                      <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
+                        <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0 rounded-s-md`}></div>
+                        <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
+                          <span className="font-normal pr-4">Subject:</span> {data.sessionDetail.meta_data.subject ?? "Science"}
+                          <div className="text-sm md:text-base font-semibold ">
+                            <span className="font-normal pr-7">Batch:</span> {data.sessionDetail.meta_data.batch ?? "Master Batch"}
+                          </div>
                         </div>
+                        {renderButton(data)}
                       </div>
-                      {renderButton(data)}
                     </div>
-                  </div>
-                ))}
-              </div>) : (
-              <p className={infoMessageClass}>
-                Good Job! There are no more pending live classes today.
-              </p>
-            )}
-          </div>
-          <div className="bg-heading">
-            <h1 className="text-primary ml-4 font-semibold text-xl">Tests</h1>
-            {quizzes.length > 0 ? (
-              <div className="grid grid-cols-1 gap-4 pb-40">
-                {quizzes.map((data, index) => (
-                  <div key={index} className="flex mt-4 items-center" >
-                    <div>
-                      <p className={`${commonTextClass}`}>
-                        {formatSessionTime(data.sessionOccurrence.start_time)}
-                      </p>
-                      <p className={`${commonTextClass}`}>
-                        {formatSessionTime(data.sessionOccurrence.end_time)}
-                      </p>
-                    </div>
-                    <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
-                      <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0  rounded-s-md`}></div>
-                      <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
-                        <span className="font-normal pr-4">Subject:</span> {data.sessionDetail.meta_data.stream}
-                        <div className="text-sm md:text-base font-semibold ">
-                          <span className="font-normal pr-9">Type:</span> {data.sessionDetail.meta_data.test_type}
+                  ))}
+                </div>) : (
+                <p className={infoMessageClass}>
+                  Good Job! There are no more pending live classes today.
```

Comment:
Why "Good job!"? :D 

Instead, maybe say "There are no more live classes today. You can relax" :)

Reply:
Poojita has included these texts in the mockup XD. Can I confirm with her?

Reply:
Changed!